Sources/Commands/SwiftTestTool.swift

```diff
@@ -488,14 +488,21 @@ final class ParallelTestRunner {
             thread.start()
             return thread
         })
-
-        // Holds the output of test cases which failed.
+        
+        // Whether to output on success
+        let shouldOutputSuccess = Process.verbose
```

Comment:
I think we should make this a computed property of the ParallelTestRunner class.